crates/holochain_sqlite/src/schema/p2p/insert.sql

```diff
@@ -0,0 +1,21 @@
+-- because UPSERT isn't guaranteed to exist on our sqlite version
```

Comment:
why not? what would need to happen to make it so we can upsert?

Reply:
we would need the bundled sqlcipher to land: https://github.com/rusqlite/rusqlite/issues/765